playbooks/ansible-changelog-fragment/run.yaml

```diff
@@ -2,14 +2,16 @@
 - hosts: localhost
   gather_facts: false
   tasks:
-    - name: Check for changelog fragments
-      args:
-        chdir: "{{ zuul.executor.src_root }}/{{ zuul.project.canonical_name }}"
-      shell: git show --name-status --exit-code --oneline --diff-filter=A | grep changelogs/fragments/
-      register: r
-      failed_when: r.rc > 1
-
-    - name: Changelog fragment failed
-      fail:
-        msg: "Your pull-request is missing a changelog fragment, please add one. It should explain to end users the reason for your change."
-      when: r.rc
+    - when: zuul.project.canonical_hostname == "github.com"
+      block:
+        - name: Retrieve the details about the Github project
+          uri:
+            url: "https://api.github.com/repos/{{ zuul.project.name }}"
```

Comment:
@pabelanger, @quidame this is how I retrieve the `default-branch`. Zuul does not expose the information as far as I can see.